src/Uno.UI.Wasm/tsBindings/WindowManagerRegisterUIElementParams.ts

```diff
@@ -0,0 +1,61 @@
+/* TSBindingsGenerator Generated code -- this code is regenerated on each build */
+class WindowManagerRegisterUIElementParams
+{
+	/* Pack=4 */
+	TypeName : string;
+	IsFrameworkElement : boolean;
+	Classes_Length : number;
+	Classes : Array<string>;
+	public static unmarshal(pData:number) : WindowManagerRegisterUIElementParams
+	{
+		let ret = new WindowManagerRegisterUIElementParams();
+		
+		{
+			var ptr = Module.getValue(pData + 0, "*");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'ptr' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)